spec/listeners/action_cable_listener_spec.rb

```diff
@@ -149,7 +149,7 @@
     end
 
     it 'broadcast event with label data' do
-      expect(conversation.reload.push_event_data[:labels]).to eq(conversation.labels.pluck(:id, :name))
+      expect(conversation.reload.push_event_data[:labels]).to eq(conversation.labels.pluck(:name))
```

Comment:
Shouldn't this be conversation.label_list?

Reply:
@pranavrajs `label_list` is what we are fetching in the `push_event_data['labels']` and this gives more idea of what it contains.